doc/api/http.md

```diff
@@ -56,13 +56,13 @@ The HTTP Agent also defaults client requests to using
 socket to become free the socket is closed. This means that Node.js's
 pool has the benefit of keep-alive when under load but still does not
 require developers to manually close the HTTP clients using
-KeepAlive.
+keepAlive.
 
-If you opt into using HTTP KeepAlive, you can create an Agent object
+If you opt into using HTTP keepAlive, you can create an Agent object
```

Comment:
@lance I think that's typically only done if you were referencing a module other than the one the current documentation is for.